src/modules/account/utils/logout.js

```diff
@@ -1,5 +1,17 @@
-export default function logout(){
-    //should expire cookie 
-    // remove the login data from local storage[dependent on login]
-localStorage.removeItem('user')
+import {logoutUser} from '../services'
+import getToken from '@/utils/getToken';
+export default async function logout(){
+    const token = getToken()
+    if (!token){
+        //show error message in a dialog box or a toast
+        console.log('no token found');
+        return false
+
+    }
+    const response = await logoutUser(token)
+    if (response.data.status.toLowerCase() != 'success'){
+    //use toast to show error
```

Comment:
Why there is nothing inside the if block

Reply:
We're planning on implementing toast messages for every sucess and error message on the project. So the task inside the if blaock will fall under a different card